src/stores/search/FiltersStore.ts

```diff
@@ -73,12 +73,12 @@ export class FiltersStore {
             }
 
             if (!acc[category]) {
```

Comment:
any reason to keep this dead code around?

Reply:
nope